### ISSUE TITLE
allow user to choose between INTERNAL and FILE-IO interface

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/simphony-lammps-md#54]

**Comment by nathanfranklin** [comment]
Currently we have: 
```
from simphony.engine import lammps

# defaults to FILE-IO
lammps_md_fileio =  lammps.LammpsWrapper()

# FILE-IO
lammps_md_fileio = LammpsWrapper(interface=lammps.InterfaceType.FILEIO)

#INTERNAL
lammps_md_internal = lammps.LammpsWrapper(interface=lammps.InterfaceType.INTERNAL)
```

Maybe it would be better to have a simple bool:
```
from simphony.engine import lammps

# defaults to FILE-IO
lammps_md_fileio =  lammps.LammpsWrapper()

# FILE-IO
lammps_md_fileio = LammpsWrapper(use_internal_interface=False)

#INTERNAL
lammps_md_internal = LammpsWrapper(use_internal_interface=True)
```

and make the changes suggested in #19

**Comment by mehdisadeghi** [comment]
I think it depends on how many engines we will have. If there are only FileIO and INTERNAL it makes sense to have a boolean flag to choose one over another, otherwise it is better to define interface type explicitly.

**Comment by nathanfranklin** [comment]
> I think it depends on how many engines we will have. If there are only FileIO and INTERNAL it makes sense to have a boolean flag to choose one over another, otherwise it is better to define interface type explicitly.

I completely agree.  As we have only two types (FILE-IO and INTERNAL), I will use the boolean.

**Comment by nathanfranklin** [comment]
closed by #69
